src/msw/fontdlg.cpp

```diff
@@ -49,18 +49,54 @@ wxIMPLEMENT_DYNAMIC_CLASS(wxFontDialog, wxDialog);
 // implementation
 // ============================================================================
 
+// ----------------------------------------------------------------------------
+// font dialog hook proc
+// ----------------------------------------------------------------------------
+
+static
+UINT_PTR CALLBACK
+wxFontDialogHookProc(HWND hwnd,
+                       UINT uiMsg,
+                       WPARAM WXUNUSED(wParam),
+                       LPARAM lParam)
+{
+    if ( uiMsg == WM_INITDIALOG )
+    {
+        CHOOSEFONT *pCH = (CHOOSEFONT *)lParam;
+        wxFontDialog * const
+            dialog = reinterpret_cast<wxFontDialog *>(pCH->lCustData);
+
+        const wxString title = dialog->GetTitle();
+        if ( !title.empty() )
+            ::SetWindowText(hwnd, title.t_str());
+    }
+
+    return 0;
+}
+
 // ----------------------------------------------------------------------------
 // wxFontDialog
 // ----------------------------------------------------------------------------
 
+void wxFontDialog::SetTitle(const wxString& title)
+{
+    m_title = title;
+    wxFontDialogBase::SetTitle(title);
```

Comment:
Do we really need this? I think this does nothing useful so I'd rather remove this if you don't mind (I can do it myself, no need to update your PR just for this).

Reply:
OK, thanks!